source/includes/connection-snippets/x509-snippets/nodejs-connection.js

```diff
@@ -0,0 +1,21 @@
+// begin x509 connection
+const MongoClient = require('mongodb').MongoClient;
+const fs = require('fs');
+
+const cert = fs.readFileSync("/etc/certs/mongodb/client.pem");
+const key = fs.readFileSync("/etc/certs/mongodb/client.pem");
```

Comment:
Out of curiosity is it really necessary to read in cert and key? Why not just use them both under a different name?

Reply:
It is not necessary, I don't particularly like that we're reading this same file twice - but this is consistent w/ the node-driver example for x509 (see: https://mongodb.github.io/node-mongodb-native/3.3/tutorials/connect/ssl/ > heading: "Validate Server Certificate and Present Valid Certificate")